json_tokener.c

```diff
@@ -298,11 +298,13 @@ struct json_object *json_tokener_parse_ex(struct json_tokener *tok, const char *
 	unsigned int nBytes = 0;
 	unsigned int *nBytesp = &nBytes;
 
+#ifndef __LOCALE_C_ONLY
```

Comment:
Where does `__LOCALE_C_ONLY` get set?  I would expect to see something in CMakeLists.txt